.vscode/settings.json

```diff
@@ -12,5 +12,6 @@
   "[dockerfile]": {
     "editor.formatOnSave": false
   },
-  "python.analysis.extraPaths": ["./lib", "./tests"]
+  "python.analysis.extraPaths": ["./lib", "./tests"],
+  "python.languageServer": "None"
```

Comment:
Why changing this?

Reply:
Sorry, it was done automatically by my vscode. I think that this PR is ready to merge, as the tests have passed before this last change.